dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1662,7 +1662,7 @@ a.download-video {
   overflow: hidden;
 }
 
-.text-match, .free-response {
```

Comment:
Is there a benefit to moving it to `code-studio`?  I don't expect it to be used outside of Dashboard.